docs/2.7.0/docs/canton/usermanual/security.rst

```diff
@@ -641,6 +641,44 @@ be aware that using the key id is not enough and we are required to register the
 Finally, we need to initialize our :ref:`domain <manually-init-domain>` and
 :ref:`participants <manually-init-participant>` using the previously registered keys.
 
+.. _live_provider_migration:
+
+Live Provider Migration
```

Comment:
```suggestion
Participant Node Migration to KMS Crypto Provider
```

Reply:
"live" suggests that this can be done online without disruption, which is not true.